test/Robot/YouBotsGripping.jl

```diff
@@ -559,7 +559,7 @@ stopTime = 28.0
 tolerance = 1e-6
 if simplifiedContact
     if Sys.iswindows()
-        requiredFinalStates = [0.005097042296241514, 0.7904195393215345, 0.1841940493955645, 8.446317251467069e-12, -2.2576132118717483e-11, -1.7303680200672947e-13, -1.4034248291585147, 1.0539134148527454, 0.3556479379736642, 1.4837457101047398e-10, -1.5860016149510553e-12, 8.466449306855364e-10, -5.035709748313884e-13, 5.038211504608792e-13, -6.4236559356515524e-12, 6.4244774123766845e-12, -1.4666749065144926e-11, 1.466863501281412e-11, -9.012353780840725e-12, 9.013783071647532e-12, 9.032593402455524e-14, -9.033163022595203e-14, -3.8473454247796905e-9, 0.2416353232213723, -0.0007949230994956009, -0.0009396161976279393, 3.142237057380001e-10, -1.0374028776592695e-8, 0.0009999997883335914, 2.1161891973887514e-10, 4.7611903716442403e-7, -4.757994858867564e-7, -4.541716958187125e-6, 4.5422655349058656e-6, -8.529075558084379e-6, 8.530276124760662e-6, -5.053507452926378e-6, 5.054369332505813e-6, 3.5345478895757224e-8, -3.5348923488087324e-8, 0.0036406708453183144, 0.20692244111073219, -0.04258153403369738, -0.018517682925623752, 0.00012295471350878592, -0.0014251993197977888, 0.0009709209574487991, 2.9072761454933187e-5]
+        requiredFinalStates = [0.0050970438521427686, 0.7904209132350711, 0.18419404940475442, -1.8137251625912176e-16, 5.15723222832311e-16, 4.3563251639972627e-16, -1.4095490501776007, 1.052597940983404, 0.3609484049540662, -3.643852247439651e-15, 2.7004689815539263e-16, -1.898293154574278e-14, -5.035685853449397e-13, 5.038187563005594e-13, -6.423546330581647e-12, 6.424367473457469e-12, -1.466646015541777e-11, 1.46683460703095e-11, -9.012171586569444e-12, 9.013600853782148e-12, 9.032215135499194e-14, -9.032784745301517e-14, -3.847327194783694e-9, 0.24163532322220996, -0.0007949230980801731, -0.0009396161969942047, 3.1421054610143916e-10, -1.0373829067802092e-8, 0.0009999997883376663, 2.1161485005970094e-10, 4.7610113997790455e-7, -4.758629591336826e-7, -4.541758289187212e-6, 4.54228619794064e-6, -8.529188503662272e-6, 8.530367912683205e-6, -5.053575394379844e-6, 5.0544367215036136e-6, 3.5345587109034675e-8, -3.5349028754053165e-8, 0.003640521453754823, 0.20692212203816962, -0.042582089475276606, -0.018517919284936582, 0.00012295508999650139, -0.001425191900008411, 0.0009709211088039017, 2.907249543964317e-5]
```

Comment:
Why do results change?

Reply:
I don't know. I changed from `rot1, rot3, rot123` to vectors like I did for the other robot models (the same values are used there). 
I ran this model several times. New final states are required only if I change `rot3` to vector in line 430.

Reply:
I checked this:

```
julia> Modia3D.rot3(-180u"°")
3×3 SMatrix{3, 3, Float64, 9} with indices SOneTo(3)×SOneTo(3):
 -1.0  -0.0  0.0
  0.0  -1.0  0.0
  0.0   0.0  1.0

julia> Modia3D.rot3(-pi)
3×3 SMatrix{3, 3, Float64, 9} with indices SOneTo(3)×SOneTo(3):
 -1.0          -1.22465e-16  0.0
  1.22465e-16  -1.0          0.0
  0.0           0.0          1.0

```